lnbits/extensions/boltz/boltz.py

```diff
@@ -0,0 +1,458 @@
+import os
+import asyncio
+import json
+import httpx
+import calendar
+import datetime
+from starlette.exceptions import HTTPException
+from http import HTTPStatus
+from lnbits import bolt11
+from websockets import connect
+from typing import Union
+from embit import ec, script
+from embit.networks import NETWORKS
+from embit.transaction import Transaction, TransactionInput, TransactionOutput, SIGHASH
+from binascii import unhexlify, hexlify
+from lnbits.helpers import urlsafe_short_hash
+from loguru import logger
+
+from hashlib import sha256
+
+from lnbits.core.services import (
+    get_wallet,
+    fee_reserve,
+    check_invoice_status,
+    create_invoice,
+    create_payment,
+    delete_payment,
+    pay_invoice,
+)
+
+from .crud import update_swap_status
+
+from .models import (
+    CreateSubmarineSwap,
+    SubmarineSwap,
+    CreateReverseSubmarineSwap,
+    ReverseSubmarineSwap,
+)
+
+from lnbits.settings import DEBUG
+
+if DEBUG:
+    net = NETWORKS["regtest"]
+    BOLTZ_URL = "http://boltz:9001"
+    MEMPOOL_SPACE_URL = "http://mempool-web:8080"
+    MEMPOOL_SPACE_URL_WS = "ws://mempool-web:8080"
+else:
+    net = NETWORKS["main"]
+    BOLTZ_URL = "https://boltz.exchange/api"
+    MEMPOOL_SPACE_URL = "https://mempool.space"
+    MEMPOOL_SPACE_URL_WS = "wss://mempool.space"
```

Comment:
Are these supposed to be set by the user? 

If so, maybe the string should be checked for no trailing `/` since the rest of the code wouldn't work otherwise.

Reply:
i guess, they could be set, but not now for this release, i am planning a mempool space extension, so not everyone needs to handle mempool stuff.